lib/challenge.js

```diff
@@ -30,10 +40,14 @@
   // - a name
   // - an address
 
-var pet_owner;
 
-// Answer Starts Here
 
+// Answer Starts Here
+var pet_owner {
```

Comment:
You need an assignment operator (`=`) here between `pet_owner` and `{`. As the code here stands, it will throw a syntax error because the interpreter doesn't know what to do with an object literal immediately following a variable declaration.